packages/engine/src/avatar/systems/AvatarAnimationSystem.ts

```diff
@@ -173,7 +173,7 @@ const execute = () => {
     if (!rig?.hips?.node) continue
 
     const rigidbodyComponent = getOptionalComponent(entity, RigidBodyComponent)
-    if (rigidbodyComponent) {
+    if (rigidbodyComponent && rigidbodyComponent.body.isEnabled()) {
```

Comment:
what exactly does isEnabled() mean in this context?

Reply:
The rigidbody is disabled when the avatar is mounted. The logic below should not be run in that case. My thought process for disabling the rigidbody when mounted is to prevent conflicts when a mount point is inside of another rigidbody (like a car). This could also be done by setting collision layers, any thoughts?

Reply:
ah interesting. yes disabling seems fine for now then